mautrix_telegram/__main__.py

```diff
@@ -162,6 +165,13 @@ def is_bridge_ghost(self, user_id: UserID) -> bool:
     async def count_logged_in_users(self) -> int:
         return len([user for user in User.by_tgid.values() if user.tgid])
 
+    def update_bridge_readiness(self):
```

Comment:
Does the bridge not separate "health" and "ready"? If it does, this seems like the wrong check to adjust, if it doesn't, then I guess it doesn't matter:

https://medium.com/@AADota/kubernetes-liveness-and-readiness-probes-difference-1b659c369e17

However, what concerns me is that this will make readiness also depend on for example remote network issues. But I guess that can't be avoided.

Reply:
We do have a `live` health endpoint, that might be better

Reply:
Changed to /live in https://github.com/vector-im/mautrix-telegram/pull/23/commits/e8cff5057b5bf64c2c31ec516449732f3e406466